Q1DWeighted SANS improvement (#32061)
mantidproject/mantid: docs/source/algorithms/Q1DWeighted-v1.rst

each wavelength bin of each detector pixel, determining its Q-value, and adding its amplitude
:math:`I` to the appropriate Q bin. The algorithm works both for monochromatic and TOF SANS data.

Note that in TOF case, the algorithm performs averaging in 2 steps.
First, for each wavelength slice of the input it performs azimuthal averaging in radial rings.
This results in a stack of I(Q)s, one for each input wavelength.
As a second step, all the I(Q)s corresponding to different wavelengths are averaged again.
Note that for the TOF data, this is mathematically not equivalent to performing averaging with one step; that is averaging everything into a single I(Q) histogram.
The latter is done in :ref:`Q1D <algm-Q1D>` algorithm.
See the :ref:`Rebin <algm-Rebin>` documentation for details about choosing the ``OutputBinning`` parameter.

> For monochromatic data, the algorithm considers each bin as belonging to a different sample, and averages in distinct bins that are not merged.

Reviewer: ```suggestion
For monochromatic data, the algorithm considers each bin as belonging to a different sample and/or kinetic frame.
```

I/O
---

> The input to this algorithm must be a histogram with common wavelength bins for each pixel, or a monochromatic workspace with different sample in each bin.

Reviewer: ```suggestion
The input to this algorithm must be a histogram with common wavelength bins for each pixel, or a monochromatic workspace with different sample or kinetic frame in each bin. In this case the workspace must not be in units of wavelength, but it must have the wavelength present in the sample logs.
```
